packages/docprovider/src/yprovider.ts

```diff
@@ -22,6 +28,19 @@ export class WebSocketProviderWithLocks extends WebsocketProvider {
     super(options.url, options.guid, options.ymodel.ydoc, {
       awareness: options.ymodel.awareness
     });
+    const color =
+      '#' + env.getParam('--usercolor', getRandomColor().color.slice(1));
```

Comment:
I have not used `lib0/environment` before, just wondering, is this helping the params to persist over refreshes in the browser?

Reply:
`lib0/environment` provides some helper functions to parse url parameters. It does not persist the variables across sessions. After the discussion yesterday, I think it would be great to store the variables in `localstorage`. Is that something you'd like to work on?

Reply:
Thanks @dmonad, for the explaining ` lib0/environment` and following up the discussion yesterday. I'd be happy to give this a try.